pages/common/sftp.md

```diff
@@ -0,0 +1,20 @@
+# sftp
+
+> Secure file transfer program.
+> Copy files between hosts using Secure File Transfer Protocol over SSH.
```

Comment:
Shouldn't the page somehow state how `sftp` starts an interactive session? I usually use sftp instead of scp for just this reason.